docs/source/hardware/hardware_overview.md

```diff
@@ -143,4 +152,20 @@ widths: auto
   - 3(*x*8)
 :::
 
+:::{note}
+The a100 GPUs come in two variants, 41 GB and 82 GB memory versions. If your job requires the 82 GB version, please include the constraint flag in your job:
```

Comment:
I'd change the text to the following:

The a100 GPUs come in two variants, 41 GB and 82 GB memory versions. If your job requires the 82 GB version, please include the `--constraint=` flag in your job:

for consistency with the language above in cpu section

Reply:
Absolutely and done!!